### ISSUE TITLE
Fetch license data from the /license endpoint instead of the UI configuration

### ISSUE THREAD

**Issue by buccarel** [camunda/camunda#22294]
### Context

Currently, Optimize does not have access to the Unified REST API that's available to the rest of the stack. To compensate for the lack of this API, the pull request https://github.com/camunda/camunda/pull/21199  added two fields, to the Optimize's `ui-configuration`, that are related to the licenses.

Once Optimize is part of the single application, it will have access to the Unified REST API and to the `/license` endpoint, therefore there will be no need for the extended `ui-configuration`.

**On Hold**: this task is currently blocked by https://github.com/camunda/camunda/issues/22271

### Acceptance Criteria

[ ] The extra fields added to `ui-configuration` are removed
[ ] The data is now fetched by the front-end by calling the now-accessible `/liencse` endpoint

### Breakdown

```[tasklist]
### Pull Requests
```

## Task Lifecycle

For managing the issue lifecycle, please use the workflow commands. You can see the available
commands by writing `/help` as a comment on this issue.

#### Engineering Review

- [ ] All code targeting the main branch has been reviewed by at least one Engineer
- [ ] The PR targeting the main branch has been approved by the reviewing Engineer
- [ ] If the API has changed, the API documentation is updated
- [ ] All other PRs (docs, controller etc.) in the breakdown have been approved

#### QA Review

- [ ] The change is implemented as described on all target environments/versions/modes
- [ ] The documentation changes are as expected

### Completion

- [ ] All Review stages are successfully completed
- [ ] All associated PRs are merged to the main branch(es) and stable branches
- [ ] The correct version labels are applied to the issue

**Comment by RomanJRW** [comment]
Closing as this is no longer intended with Optimize not in the single application
